src/llmcompressor/modifiers/modifier.py

```diff
@@ -255,7 +258,7 @@ def on_finalize(self, state: State, **kwargs) -> bool:
         :return: True if the modifier was finalized successfully,
             False otherwise
         """
-        raise NotImplementedError()
+        return True
```

Comment:
I think if it is an abstract method, it is good practice to raise NotImplementedError. 
If not abstract method, then having default logic is ok

Reply:
Agree

Reply:
I actually think that this is a github diff bug. This has already been changed in the branch: https://github.com/vllm-project/llm-compressor/blob/kylesayrs/modifier-missing-funcs/src/llmcompressor/modifiers/modifier.py

Reply:
Very strange, `git diff` also shows this change. I'll need to investigate this further

Reply:
Tried changing base, dummy commits, merging with main. This bug is also seen when running `git diff main kylesayrs/modifier-missing-funcs`. It's a somewhat known bug from what I've heard

Reply:
Looks like when the merge occurred, github ignored the incorrect diff you see here and used the correct changes: https://github.com/vllm-project/llm-compressor/blob/main/src/llmcompressor/modifiers/modifier.py#L249